source/mainnet/net/mobile-wallet-gen2/setup-mobile-wallet.rst

```diff
@@ -87,9 +87,57 @@ Once the identity provider has verified your identity you can submit an account.
    .. image:: ../images/mobile-wallet-gen2/submit-acct.png
       :width: 25%
 
-.. Note::
-   To access the **Balance** of the new account, tap the Balance area on the account card or tap |morepage|.
+Basic navigation
+================
 
-.. |morepage| image:: ../images/more-ellipsis.png
-             :alt: Three dots button
-             :width: 40px
+In the wallet you have some navigation options.
+
+.. image:: ../images/mobile-wallet-gen2/main-nav.png
+      :alt: main wallet screen with position numbers
+      :width: 50%
+
+1. Balances: shows balances of all accounts.
+2. Account: shows the identity on which the account was created, and totals.
+3. :ref:`Wallet settings<mw-gen2-wallet-settings>`: contains actions for the wallet.
+4. Scan QR code: tap to :ref:`scan a QR code to connect to a dApp<connect-app-bw>`.
+5. Add: to :ref:`add a new account<create-account>`.
+6. Account actions: actions you can perform on the account, such as send, receive, bake/delegate.
```

Comment:
A range of actions. I guess there will be more actions in the account details page